src/clarity-angular/utils/_maps.clarity.scss

```diff
@@ -0,0 +1,590 @@
+    large:(),
+    xlarge:()
+);
+$clr-p1: (
+    clr-common: (
+        color: $clr-p1-color,
+        font-weight: $clr-typography-regfont,
+        font-size: 0.583333rem,
+        letter-spacing: $clr-kerning-lotsatext,
+        line-height: 1rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p2: (
+    clr-common: (
+        color: $clr-p2-color,
+        font-weight: $clr-typography-sboldfont,
+        font-size: $clr-typography-smalltext,
+        letter-spacing: $clr-kerning-smalltext,
+        line-height: 1rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p3: (
+    clr-common: (
+        color: $clr-p3-color,
+        font-weight: $clr-typography-regfont,
+        font-size: $clr-typography-smalltext,
+        letter-spacing: $clr-kerning-smalltext,
+        line-height: 1rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p4: (
+    clr-common: (
+        color: $clr-p4-color,
+        font-weight: $clr-typography-boldfont,
+        font-size: $clr-typography-xsmalltext,
+        letter-spacing: $clr-kerning-smalltext,
+        line-height: 1rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p5: (
+    clr-common: (
+        color: $clr-p5-color,
+        font-weight: $clr-typography-regfont,
+        font-size: $clr-typography-xsmalltext,
+        letter-spacing: $clr-kerning-smalltext,
+        line-height: 1rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p6: (
+    clr-common: (
+        color: $clr-p6-color,
+        font-weight: $clr-typography-boldfont,
+        font-size: $clr-typography-xxsmalltext,
+        letter-spacing: $clr-kerning-tinytext,
+        line-height: 0.5rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p7: (
+    clr-common: (
+        color: $clr-p7-color,
+        font-weight: $clr-typography-regfont,
+        font-size: $clr-typography-xxsmalltext,
+        letter-spacing: $clr-kerning-tinytext,
+        line-height: 0.5rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+$clr-p8: (
+    clr-common: (
+        color: $clr-p8-color,
+        font-weight: $clr-typography-regfont,
+        font-size: $clr-typography-tinytext,
+        letter-spacing: $clr-kerning-tinytext,
+        line-height: 0.5rem,
+        margin-top: 1rem,
+        margin-bottom: 0
+    ),
+    small:(),
+    medium:(),
+    large:(),
+    xlarge:()
+);
+//Element Properties Map
+$clr-elements: (
+    h1: $clr-h1,
+    h1-dark: #clr-h1-dark,
```

Comment:
You mean `$clr-h1-dark`?

Reply:
Nice catch, this isn't needed anymore. I will remove.